app/models/case_court_report.rb

```diff
@@ -21,11 +21,14 @@ def generate_to_string
   private
```

Comment:
test failure looks like a flake
> 1) Seeds test development DB successfully populates all necessary tables
     Failure/Error: ActiveRecord::Tasks::DatabaseTasks.load_seed

     ActiveRecord::RecordInvalid:
       Validation failed: Volunteer and case must belong to the same organization

Reply:
I've seen that in least least one other pr